### DISCUSSION TITLE
How much fil will be awarded to the next block and how to calculate it? Now it is about 12.7869 FIL, thank you

### DISCUSSION THREAD

**Discussion by tzj1042** [filecoin-project/lotus#4842] [Storage Provider] [answered]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

**Describe the problem**

A brief description of the problem you encountered while proving (sealing) a sector.

Including what commands you ran, and a description of your setup, is very helpful.

**Sectors status**

The output of `lotus-miner sectors status --log <sectorId>` for the failed sector(s).

**Lotus miner logs**

Please go through the logs of your miner, and include screenshots of any error-like messages you find.

Alternatively please upload full log files and share a link here

**Lotus miner diagnostic info**

Please collect the following diagnostic information, and share a link here

* lotus-miner diagnostic info `lotus-miner info all > allinfo`

** Code modifications **

If you have modified parts of lotus, please describe which areas were modified,
and the scope of those modifications

**Version**

The output of `lotus --version`.

**Comment by jennijuju** [comment]
Questions should be asked in discussion.

**Reply by tzj1042** [reply]
Hello, I want to know how to calculate 12.7FIL of a block reward?

**Comment by jennijuju** [comment, marked as answer]
https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/reward/reward_actor.go

**Comment by tzj1042** [comment]
thank you



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Jennifer"<notifications@github.com&gt;; 
发送时间: 2020年11月13日(星期五) 下午2:14
收件人: "filecoin-project/lotus"<lotus@noreply.github.com&gt;; 
抄送: "唐宗君"<739221814@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [filecoin-project/lotus] How much fil will be awarded to the next block and how to calculate it? Now it is about 12.7869 FIL, thank you (#4842)





 
https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/reward/reward_actor.go

**Comment by tzj1042** [comment]
Hello, I want to know how to calculate 12.7 of a block reward?



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Jennifer"<notifications@github.com&gt;; 
发送时间: 2020年11月13日(星期五) 下午2:14
收件人: "filecoin-project/lotus"<lotus@noreply.github.com&gt;; 
抄送: "唐宗君"<739221814@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [filecoin-project/lotus] How much fil will be awarded to the next block and how to calculate it? Now it is about 12.7869 FIL, thank you (#4842)





 
https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/reward/reward_actor.go
